djweb/dj_comp_hist/views.py

```diff
@@ -163,11 +163,11 @@ def search_results(request):
     folder_objs = Folder.objects.filter(full__contains=user_input)
     organization_objs = Organization.objects.filter(Q(name__contains=user_input)|Q(
         location__contains=user_input))
-    doc_type = ["minutes", "memo", "proposal", "letter", "receipt", "contract", "notice",
```

Comment:
Not too important, but is there a reason you deleted all the spaces?  I thought the standard was to put spaces between elements of lists

Reply:
i think this just happened because of pulling & merging from master branch